packages/@o3r/design/src/core/design-token/design-token-specification.interface.ts

```diff
@@ -290,8 +288,7 @@ export const isDesignTokenGroup = (node?: any): node is DesignTokenGroup => {
  * @param value Stroke Style value
  * @returns true if it is a defined value
  */
-// eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents
-export const isTokenTypeStrokeStyleValueComplex = (value?: DesignTokenTypeStrokeStyleValue | string): value is DesignTokenTypeStrokeStyleDetailsValue => {
+export const isTokenTypeStrokeStyleValueComplex = (value?: DesignTokenTypeStrokeStyleValue): value is DesignTokenTypeStrokeStyleDetailsValue => {
```

Comment:
```suggestion
export const isTokenTypeStrokeStyleValueComplex = (value?: Exclude<DesignTokenTypeStrokeStyleValue, string> | string): value is DesignTokenTypeStrokeStyleDetailsValue => {
```
else we could not call this method `isTokenTypeStrokeStyleValueComplex('my-custom-string-not-matching-string-in-DesignTokenTypeStrokeStyleValue')